### PR TITLE
Split QAQC into separate bash script

### PR DESCRIPTION
**Pull request by SashaWeinstein**
Simple refactor, one reviewer 🌻 
Addresses issue #205 
Will hopefully make it easier to add the new queries @Oysters1874 wrote about in this [COLP QAQC brainstorm](https://github.com/NYCPlanning/data-engineering-qaqc/issues/164) issue

### CODE REVIEW

**Review by Oysters1874 — APPROVED** [review]
lgtm

**Comment by mbh329** [comment]
This didn't work locally on my machine

File: bash/02_build.sh [hunk 1]
```diff
@@ -11,16 +11,14 @@ docker run --rm\
...
     -u $(id -u ${USER}):$(id -g ${USER}) \
     -e BUILD_ENGINE=$BUILD_ENGINE\
-    nycplanning/docker-geosupport:latest bash -c "python3 -m python.geo_qaqc;
-                                                python3 -m python.geocode"
+    nycplanning/docker-geosupport:latest bash -c "python3 -m python.geocode"
 
 psql $BUILD_ENGINE -f sql/_procedures.sql
 psql $BUILD_ENGINE -f sql/clean_parcelname.sql
 psql $BUILD_ENGINE -f sql/create_colp.sql
-psql $BUILD_ENGINE -f sql/geo_qaqc.sql
```
Comment (mbh329) [inline bash/02_build.sh:20]: I think moving this out of the build step creates an issue on my local machine @SashaWeinstein. wanna swing by when you get a chance?

**Review by mbh329 — CHANGES_REQUESTED** [review]

**Comment by mbh329** [comment]
Local build worked 2nd time around.lgtm

**Review by mbh329 — APPROVED** [review]

**Pull request merged**